src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkCreationService.java

```diff
@@ -162,7 +162,7 @@ BulkCasesPayload updateBulkRequest(BulkRequest bulkRequest, String authToken, bo
                 errors.addAll(bulkCasesPayload.getErrors());
             }
         } catch (Exception ex) {
-            log.error("Error processing bulk update threads");
+            log.error("Error processing bulk update threads:" + ex.toString());
```

Comment:
You should usually include the exception as an argument to log.error so that the full stacktrace will be output
`log.error("Error processing bulk update threads: " + ex.getMessage(), ex);`

Reply:
refactored